src/easylink/configuration.py

```diff
@@ -122,10 +118,7 @@ def slurm(self) -> Dict[str, Any]:
 
     @property
     def spark(self) -> Dict[str, Any]:
```

Comment:
we determine whether to do anything with the spark configuration in the PipelineGraph (when we know what all the validated implementations are), so in the config I'm just letting use keep the spark defaults in even if they're not necessarily used.